fe/fe-core/src/main/java/org/apache/doris/planner/SingleNodePlanner.java

```diff
@@ -1031,9 +1031,33 @@ private PlanNode createSelectPlan(SelectStmt selectStmt, Analyzer analyzer, long
 
             for (int i = 1; i < selectStmt.getTableRefs().size(); ++i) {
                 TableRef innerRef = selectStmt.getTableRefs().get(i);
+                boolean aggOutputNotHaveMaterializedSlot = false;
+                AggregateInfo aggregateInfo = null;
+                TupleDescriptor output = null;
+                if (innerRef instanceof InlineViewRef) {
+                    InlineViewRef inlineViewRef = (InlineViewRef) innerRef;
+                    QueryStmt queryStmt = inlineViewRef.getViewStmt();
+                    if (queryStmt instanceof SelectStmt) {
```

Comment:
set operation do not has this bug?

Reply:
I think this bug is just about query block, no matter the stmt of origin SQL is SetOperationStmt or SelectStmt